packages/controllers/src/snaps/SnapController.ts

```diff
@@ -927,12 +935,14 @@ export class SnapController extends BaseController<
    *
    * @param snapId - The id of the snap to check.
    * @param version - The version to check.
+   * @param sourceShaSum - Shasum of the snap's source code.
    */
   private async _assertIsUnblocked(
     snapId: ValidatedSnapId,
     version: SemVerVersion,
+    sourceShaSum: string,
```

Comment:
Perhaps this should be an options bag now with named parameters instead of positional?